pages/api/users.ts

```diff
@@ -0,0 +1,17 @@
+import nextConnect from 'next-connect';
+import { NextApiRequest, NextApiResponse } from 'next';
+import middleware from '../../middlewares/middleware';
+import { Document } from 'mongoose';
+import User from '../../models/user/user.schema';
+
+const handler = nextConnect();
+handler.use(middleware);
+
+handler.get(
```

Comment:
dla tych wszystkich plików w `/api` i `/pages` przyda się jeszcze obsługa błędów, tak aby w razie problemów z połączeniem z DB, nie było błędów i żeby nie wywalało strony.

wiem, że w moim przykładzie nie było nic takiego, ale to była prowizorka trochę :)

Reply:
zrobione - mam nadzieję, że tak jak trzeba.